smap/sources/conf/twitter_conf.ini

```diff
@@ -0,0 +1,14 @@
+[server]
+Port = 8002
+
+[/]
+uuid = 6fae1782-27b2-12e1-2a48-370ecdac5e01
+
+[/twitter]
+type = twitterDriver.TwitterDriver
+Rate = 43200
+consumer_key = 5EvNnwcByDSdb0rxiW5J0mKaT
+consumer_secret = kBU9LfH099KajuZiRJasYJWI6agX6J0CViTY4QciRptLpfoFPX
```

Comment:
The idea is to have master copy and server has the master git repository. Having a internal repo is a great idea. But then how we manage two git repos?